react/src/pages/InfoPage.js

```diff
@@ -227,21 +236,37 @@ const InfoPage = () => {
             <InfoRow name={"Endpoint"} value={info.get("page.endpoint")} monospace={true} size="2" />
             <InfoRow name={"Client (React UI)"} value={Client.BaseUrl()} monospace={true} size="2" />
             <InfoRow name={"Server (React API)"} value={Server.BaseUrl()} monospace={true} size="2" />
+            <InfoRow name={"Checks File"} value={info.data?.checks?.file} monospace={true} size="2" />
+            { header.app?.accounts &&
+                <InfoRow name={"Accounts File"} value={header.app?.accounts} monospace={true} size="2" />
+            }
         </InfoBox>
-        <InfoBox title={`GAC: ${info.get("gac.name")}`}>
+        <InfoBox title={`GAC: ${info.get("gac.name")}`} show={false}>
             { info.get("gac.values") ? (<span>
                 { Object.keys(info.get("gac.values")).map((key) => {
                     return <InfoRow key={key} name={key} value={info.get("gac.values")[key]} monospace={true} copy={true} />
                 })}
             </span>):(<span/>)}
         </InfoBox>
-        <InfoBox title="Environment Variables">
+        <InfoBox title="Environment Variables" show={false}>
             { info.get("environ") ? (<span>
                 { Object.keys(info.get("environ")).map((key) => {
                     return <InfoRow key={key} name={key} value={info.get("environ")[key]} monospace={true} copy={true} />
                 })}
             </span>):(<span/>)}
         </InfoBox>
+        { header.app?.accounts &&
+            <div className="container" style={{marginTop:"4pt"}}>
+                { showingAccounts ? <>
+                    <AccountsComponent />
+                </>:<>
+                    <b onClick={() => setShowingAccounts(true)} style={{cursor:"pointer"}}>Show Accounts</b>
+                    <div className="box">
+                        Click <b onClick={() => setShowingAccounts(true)} style={{cursor:"pointer"}}>here</b> to <span onClick={() => setShowingAccounts(true)} style={{cursor:"pointer"}}>show</span>.
```

Comment:
Here you're marking up a `<b>` and elsewhere a `<span>` for essentially the same thing, the word "here". Some consistency might be good so that the style rules are kept to a minimum but also changing one rule doesn't just change half of things and not the other half.

Reply:
Yeah, lots of places where I would like to and eventually will factor out to common/shared CSS classes and/or whatnot.